### ISSUE TITLE
 r2dbc-h2 Doesn't it provide webconsole ?

### ISSUE THREAD

**Issue by DongJu-Na** [r2dbc/r2dbc-h2#239]
```
  @Bean
  @Override
  public ConnectionFactory connectionFactory() {
    ConnectionFactoryOptions options = ConnectionFactoryOptions.builder()
        .option(DRIVER, "h2")
        .option(HOST, "localhost")
        .option(PROTOCOL,"mem") // mem : 인메모리 file : 로컬 파일 경로 
        .option(USER, "sa")
        .option(PASSWORD, "")
        .option(DATABASE, "r2dbc:h2:mem:///testdb")
        .option(SSL, true)
        .option(Option.valueOf("sendStringParametersAsUnicode"), false)
        .build();
    return ConnectionFactories.get(options);
  }

```

What tool should I use to verify that this code works? 🤔

**Comment by gregturn** [comment]
No, r2dbc-h2 does NOT provide a web console. It purely exists to let you write R2DBC code on top of H2, a database that essentially comes with JDBC baked into it.

If you are needing to test things, you can either use automated testing, for which we have many support tools. You can also leverage JdbcTemplate on the side to either pre-inject data, or use it after an R2DBC operation to inspect results.

Or you can perform manual operations and then use external tools, whether that is H2's own console apps or other 3rd party tools that let you examine the database.

In the future, if you have questions, you may get a faster response if you post them on stackoverflow.com and tag them "r2dbc".
